NobelLaureateDetails/Models/Details.cs

```diff
@@ -23,8 +23,10 @@ public partial class LaureateDetails
 
     public partial class Laureate
     {
+        [Required] //felt that the ID field should be mandatory
```

Comment:
Avoid putting discussion/conversation/justification in source code comments.  A good place for that is in the pull request itself.  In source code comments, it's clutter.